### PR TITLE
ffead-cpp-apache2 addition

### PR DESCRIPTION
**Pull request by nbrady-techempower**
This updates #2061 ffead update and apache test addition

### CODE REVIEW

File: frameworks/C++/ffead-cpp/benchmark_config.json [hunk 1]
```diff
@@ -23,6 +23,29 @@
       "display_name": "ffead-cpp",
       "notes": "",
       "versus": ""
+    },
+    "apache2": {
+      "setup_file": "setup",
```
Comment (sumeetchhetri) [inline frameworks/C++/ffead-cpp/benchmark_config.json]: Don't you think that this should have been setup-apache2 instead of setup??

Comment (nbrady-techempower) [inline frameworks/C++/ffead-cpp/benchmark_config.json]: oops, indeed it should.

**Comment by sumeetchhetri** [comment]
Looks all set Nate!!

**Comment by nbrady-techempower** [comment]
@sumeetchhetri it's not passing travis, do you mind taking a look when you have a moment

**Comment by sumeetchhetri** [comment]
Nate, unable to raise a PR against your forked repository, hence am raising another new PR directly against the base repo. Please close this one, I have fixed the issue in a new PR.

**Comment by sumeetchhetri** [comment]
https://github.com/TechEmpower/FrameworkBenchmarks/pull/2173

**Comment by knewmanTE** [comment]
closing in favor it #2173

**Pull request closed without merging**